components/director/pkg/formationconstraint/validation.go

```diff
@@ -198,6 +200,33 @@ func emptyNotificationStatusReturnedOperationDetails() *NotificationStatusReturn
 			Value: json.RawMessage("\"\""),
 			Error: json.RawMessage("\"\""),
 		},
+		FormationAssignmentTemplateInput: &webhook.ApplicationTenantMappingInput{
+			Operation:   model.AssignFormation,
+			FormationID: "",
+			Formation: &model.Formation{
+				ID:                  "",
+				FormationTemplateID: "",
+				Name:                "",
+			},
+			SourceApplicationTemplate: &webhook.ApplicationTemplateWithLabels{
+				ApplicationTemplate: fixApplicationTemplateModel(),
+				Labels:              fixLabels(),
+			},
+			SourceApplication: &webhook.ApplicationWithLabels{
+				Application: fixApplicationModel(),
+				Labels:      fixLabels(),
+			},
+			TargetApplicationTemplate: &webhook.ApplicationTemplateWithLabels{
+				ApplicationTemplate: fixApplicationTemplateModel(),
+				Labels:              fixLabels(),
+			},
+			TargetApplication: &webhook.ApplicationWithLabels{
+				Application: fixApplicationModel(),
+				Labels:      fixLabels(),
+			},
+			Assignment:        nil,
+			ReverseAssignment: nil,
```

Comment:
Shouldn't the `Assignment` and `ReverseAssignment` be similar to the other example where we have the `Value` and `Error` fields "zeroed"?

Reply:
We do not expect the assignments to be used from here as they are also present in the joinpoint details on top level

Reply:
Wouldnt it be better to leave them like this so that it would be explicit that we wanted it to be that way